test/box-luatest/gh_7873_func_perm_check_test.lua

```diff
@@ -0,0 +1,71 @@
+local server = require('luatest.server')
+local t = require('luatest')
+
+local g = t.group()
+
+g.before_all(function(cg)
+    cg.server = server:new()
+    cg.server:start()
+end)
+
+g.after_all(function(cg)
+    cg.server:drop()
+end)
+
+local function test_func_indices()
+    local value
+    local prefix = 'test_func_indices_'
+    local uname = prefix..'user'
```

Comment:
Usually we wrap `..` operator with spaces, like `+`. That's strange, but there are lots of unwrapped `..` in SQL test, which are under Lango team jurisdiction; perhaps they know something. But all our code usually use wrapped `..` operators.

Reply:
Fixed.